src/index.html

```diff
@@ -11,12 +11,102 @@
       content="ie=edge"
```

Comment:
The HTML-element has long attribute values. Start each one, including the first, on the new line with 2-space indentation related to the tag. Tag’s closing bracket should be on the same level as the opening one. (HTML Formatting)